core/src/impl/Kokkos_HostSpace.cpp

```diff
@@ -46,6 +46,10 @@
 #include <sstream>
 #include <cstring>
 
+#ifdef KOKKOS_COMPILER_INTEL
+#include <aligned_new>
```

Comment:
@masterleinad What was that header needed for?

Reply:
For Intel I have no idea. I only built this with GCC.

Reply:
Have a look at https://godbolt.org/z/jThT4h9Ed.

Reply:
Understood.  Thanks.